configure.ml

```diff
@@ -101,6 +101,21 @@ let find_xc_domain_save_generation_id verbose =
   Printf.printf "Looking for xc_domain_save generation_id: %s\n" (if missing then "missing" else "found");
   not missing
 
+let find_xentoollog verbose =
```

Comment:
Maybe we could make this shorter by passing `linker_flags` into the `cc` function.